src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
+
+$cellSize: 100px;
+$gap: 1px;
+$calendarSize: calc(#{$cellSize} * 7 + #{$gap} * 6);
+$monthLength: 28, 29, 30, 31;
+$startDay: (
+  "mon": 0,
+  "tue": ($cellSize + $gap),
+  "wed": 2 * ($cellSize + $gap),
+  "thu": 3 * ($cellSize + $gap),
+  "fri": 4 * ($cellSize + $gap),
+  "sat": 5 * ($cellSize + $gap),
+  "sun": 6 * ($cellSize + $gap)
+);
```

Comment:
*grammar nazi*

for collections use plural variable names:
monthLength -> monthLengths
etc